src/main/java/com/wiley/autotest/spring/SeleniumTestExecutionListener.java

```diff
@@ -593,7 +593,7 @@ private WebDriver explorer(DesiredCapabilities customDesiredCapabilities) {
     }
 
     private WebDriver explorer(String version, DesiredCapabilities customDesiredCapabilities) {
-        InternetExplorerDriverManager.getInstance().setup();
+        InternetExplorerDriverManager.getInstance().version("3.4.0").setup();
```

Comment:
there is a "version" passed as a parameter, is it correct to hardcode it like this here?